twindle-cli/renderer/epub/index.js

```diff
@@ -23,6 +27,8 @@ async function generateEpub(tweets, outputPath) {
     "Toc"
   );
 
+  console.log(threadContent);
```

Comment:
Was this console log added for testing purpose or this is required to display some message to the actual users?

Reply:
For testing purposes. Forgot to remove it. Better make it a a good first issue for beginners to solve. Will get them involved more.